examples/lujain_work_area/Contact_Book/main.py

```diff
@@ -0,0 +1,103 @@
+# main.py
+
+import csv
+import re
+from contact_book import ContactBook
+
+def read_contact_records(csv_file):
+    with open(csv_file, mode='r') as file:
+        reader = csv.DictReader(file)
+        for row in reader:
+            print(row)
+
+def validate_phone(phone):
```

Comment:
ContactRecord should be responsible for validating the data while the object is being created to assigned any data